lib/sidekiq/processor.rb

```diff
@@ -234,5 +234,14 @@ def cloned(thing)
       Marshal.load(Marshal.dump(thing))
     end
 
+    def constantize(str)
+      names = str.split('::')
+      names.shift if names.empty? || names.first.empty?
```

Comment:
Ok, thanks for merging this, looking forward to the new monkey patch free release 😜